entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+
+if [ ! -z "$CONF_URL" ] ; then
+   ENV="-env $CONF_URL"
+fi
+
+if [ ! -z "$SECRET" ] ; then
+  ENVHEADER="--env-header /run/secrets/$SECRET"
+fi
+
+dockerize \
```

Comment:
FYI Dockerize isn't just for template expansion, its intended to start the server process after waiting for dependencies to start their listeners, and then deal with logs. So you'd also put any mongod and rancher endpoint checks in here, even if you aren't going to display logs on the console with it.

Reply:
Yeah.  I don't think it is worth adding a monitor for rancher since rancher would be starting it.  And it doesn't have a database.  So I don't think there is anything more to add.